Makefile

```diff
@@ -33,8 +33,8 @@ api-docs-sphinx: swagger	## generate the sphinx source files for api-docs
 api-docs: api-docs-sphinx	## build the REST api docs
 	cd docs/api; make html
 
-feature-toggles-docs:
-	$(MAKE) -C docs/featuretoggles html
+internals-docs:
```

Comment:
I don't love this name.  It doesn't really tell me what these docs are.  Isn't everything in `docs/` _internal_? I think `guides` is more clear.  I don't have a great alternative.  What do you think of `docs/technical` (from https://en.wikipedia.org/wiki/Software_documentation)?

Reply:
> I don't love this name.

Yeah, me neither. "technical" sounds good to me.